package.json

```diff
@@ -40,23 +40,23 @@
   },
   "DEPS-NOTE": "Any non-test dependency must be added to the build_package.js script as well",
   "dependencies": {
-    "@babel/code-frame": "^7.12.13",
-    "@babel/core": "^7.14.0",
-    "@babel/plugin-proposal-class-properties": "^7.13.0",
-    "@babel/plugin-proposal-dynamic-import": "^7.13.8",
-    "@babel/plugin-proposal-export-namespace-from": "^7.12.13",
-    "@babel/plugin-proposal-logical-assignment-operators": "^7.13.8",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.13.8",
-    "@babel/plugin-proposal-numeric-separator": "^7.12.13",
-    "@babel/plugin-proposal-optional-chaining": "^7.13.12",
-    "@babel/plugin-proposal-private-methods": "^7.13.0",
-    "@babel/plugin-proposal-private-property-in-object": "^7.14.0",
+    "@babel/code-frame": "^7.14.5",
+    "@babel/core": "^7.14.8",
+    "@babel/plugin-proposal-class-properties": "^7.14.5",
+    "@babel/plugin-proposal-dynamic-import": "^7.14.5",
+    "@babel/plugin-proposal-export-namespace-from": "^7.14.5",
+    "@babel/plugin-proposal-logical-assignment-operators": "^7.14.5",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.14.5",
+    "@babel/plugin-proposal-numeric-separator": "^7.14.5",
+    "@babel/plugin-proposal-optional-chaining": "^7.14.5",
+    "@babel/plugin-proposal-private-methods": "^7.14.5",
+    "@babel/plugin-proposal-private-property-in-object": "^7.14.5",
     "@babel/plugin-syntax-async-generators": "^7.8.4",
     "@babel/plugin-syntax-json-strings": "^7.8.3",
     "@babel/plugin-syntax-object-rest-spread": "^7.8.3",
     "@babel/plugin-syntax-optional-catch-binding": "^7.8.3",
-    "@babel/plugin-transform-modules-commonjs": "^7.14.0",
-    "@babel/preset-typescript": "^7.13.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.14.5",
+    "@babel/preset-typescript": "^7.14.5",
```

Comment:
is this still there because it is used inside the test-runner? (Might make sense to use the @babel/plugin-transform-typescript directly in both? But I might also be wrong.)

Reply:
yes its still in the test runner and our webpacks. I will deal with them separately. If we want to add support for declarations to the default babel, it will technically be a breaking change. I am thinking about it.